cognite/client/experimental/time_series.py

```diff
@@ -25,6 +25,27 @@ class TimeSeriesClient(APIClient):
     def __init__(self, **kwargs):
         super().__init__(version="0.6", **kwargs)
 
+    def delete_time_series_by_id(self, ids: List[int]) -> None:
+        """Delete multiple time series by id.
+
+        Args:
+            ids (List[int]):   IDs of time series to delete.
+
+        Returns:
+            None
+
+        Examples:
+            Delete a single time series by id::
+
+                client = CogniteClient()
+
+                client.time_series.delete_time_series_by_id(ids=[my_ts_id])
+        """
+        url = "/timeseries/delete"
+        body = {"items": ids}
+        params = {}
```

Comment:
Nitpick: params can be left out. No need to pass the empty dict as the underlying requests lib will interpret None as  no parameters.

Reply:
Fair, I think I just carried that over from `get_ts_by_id`.